definitions/variable/agriculture/prices.yaml

```diff
@@ -0,0 +1,41 @@
+- Price|Agriculture|Food Products [Index]:
+    description: Weighted average price index of food crops and livestock products
+    unit: Index (2020 = 1)
+    skip-region-aggregation: true
+    weight: Agricultural Production|Crops + Agricultural Production|Livestock
```

Comment:
You have used both the attributes **weight** and **skip-region-aggregation** - in this case, the nomenclature package (i.e., the scenario processing during submission) will not compute regional aggregates and the **weight** attribute has no effect (it could be helpful for informational purposes, but could also be confusing).

Please remove the **skip-region-aggregation** attribute for all variables here, and I think it's a good compromise to use "Agricultural Production" as weight - or you could keep the **skip-region-aggregation** here (modelers can still compute and report that variable for World/R5/... themselves).